docs/src/codegen-intro.md

```diff
@@ -0,0 +1,44 @@
+---
+id: codegen-intro
+title: "Test Generator"
+---
+
+Playwright comes with the ability to generate tests out of the box and is a great way to quickly get started with testing. It will open two windows, a browser window where you interact with the website you wish to test and the Playwright Inspector window where you can record your tests, copy the tests, clear your tests as well as change the language of your tests.
+
+**You will learn**
+
+- [How to generate tests with Codegen](/codegen.md#running-codegen)
+
+
+## Running Codegen
+
+```bash js
+npx playwright codegen playwright.dev
+```
+
+```bash java
+mvn exec:java -e -Dexec.mainClass=com.microsoft.playwright.CLI -Dexec.args="codegen playwright.dev"
+```
+
+```bash python
+playwright codegen playwright.dev
+```
+
+```bash csharp
+pwsh bin\Debug\netX\playwright.ps1 codegen playwright.dev
+```
+
+Run `codegen` and perform actions in the browser. Playwright will generate the code for the user interactions. `Codegen` will attempt to generate resilient text-based selectors.
+
+<img width="1183" alt="Codegen generating code for tests for playwright.dev website" src="https://user-images.githubusercontent.com/13063165/181852815-971c10da-0b55-4e54-8a73-77e1e825193c.png" />
```

Comment:
I don't like the screenshot, we try to sell codegen and generated selectors but on the screenshot there is only one navigation.

Reply:
agreed. another one added to enhance the story